generate-templates.yaml

```diff
@@ -224,7 +238,12 @@
 
   - name: Load Fedora versions
     set_fact:
-      fedora_labels: "{{ lookup('osinfo', 'distro=fedora') |select('osinfo_active') |map(attribute='short_id') |list |sort }}"
+      fedora_labels: "{{ lookup('osinfo', 'distro=fedora') |select('osinfo_active') |map(attribute='short_id') |select('match', '^fedora\\d+$') |list |sort }}"
+
+  - name: Load Fedora import urls
+    set_fact:
+      fedora_import_urls: "{{ fedora_import_urls |default([]) + lookup('osinfo', item) |attr('image_list') |selectattr('architecture', 'eq', 'x86_64') |selectattr('format', 'in', ['raw', 'qcow2']) |map(attribute='url') |list }}"
```

Comment:
the jinja seems to work, but it is hard to read. I wonder if there is another way, e.g. in another language, to express what we like to achieve.